docs/src/main/sphinx/release/release-432.md

```diff
@@ -0,0 +1,63 @@
+# Release 432 (2 Nov 2023)
+
+:::{note}
+Release notes now include a ⚠️ symbol to highlight any changes as potentially
+breaking changes. The following changes are considered and may require
+adjustments:
+* Removal or renaming of configuration properties that may prevent startup or
+  require configuration changes
+* Changes to default values for configuration properties that may significantly
+  change the behavior of a system
+* Updates to the requirements for external systems or software used with Trino,
+  such as removal of support for an old version of a data source in a connector
+* Non-backwards compatible changes to the SPI which may require plugins to be
+  updated
+* Otherwise significant changes that requires specific attention from teams
+  managing a Trino deployment
+:::
+
+## General
+
+* Improve performance of `CREATE TABLE AS` queries that contain a redundant
```

Comment:
This should be "CREATE TABLE AS ..." or "CREATE TABLE AS ... SELECT". There's no such a thing as "CREATE TABLE AS"